tests/sandbox/headers/test_x_amz_removal.py

```diff
@@ -0,0 +1,18 @@
+import requests
+import pytest
+
+from lib import Assertions
+from lib.constants.constants import VALID_ENDPOINTS, TEST_METHODS
+
+
+@pytest.mark.sandboxtest
+@pytest.mark.parametrize("method", TEST_METHODS)
+@pytest.mark.parametrize("endpoints", VALID_ENDPOINTS)
+def test_request_with_x_amz_is_removed(nhsd_apim_proxy_url, endpoints, method):
+
+    resp = getattr(requests, method)(f"{nhsd_apim_proxy_url}/{endpoints}", headers={
+        "Accept": "*/*",
+        "Origin": "https://my.website"
```

Comment:
Looks like you've created an ORIGIN constant that could be used here, there's also a couple other instances in the tests that directly set https://my.website that could be replaced as well under cors tests if you're going this route

Reply:
Made consistent across tests